add release note entry for TINY-8929: The autocompleter `ch` configuration property has been removed. Use the `trigger` property instead. (#3146)
tinymce/tinymce-docs: modules/ROOT/pages/7.0-release-notes.adoc

As a result, users no longer experience unexpected list insertions.

=== The autocompleter `ch` configuration property has been removed. Use the `trigger` property instead.
// #TINY-8929

Previously, the `ch` configuration property was used to specify the character that would trigger the autocompleter.

This property was deprecated in link:https://www.tiny.cloud/docs/tinymce/6/6.2-release-notes/#deprecated[{productname} 6.2] and has been removed in {productname} 7.0. Instead, use the `trigger` property to specify the `string` that will trigger the autocompleter.

> If `+editor.ui.registry.addAutocompleter(name, options)+` was used  in your configuration, updating your configuration from `ch: '<string>',` to `trigger: '<string>',` is required.

Author: ```suggestion
If `+editor.ui.registry.addAutocompleter(name, options)+` was used in your configuration, updating your configuration from `ch: '<string>',` to `trigger: '<string>',` is required.
```

> This property was deprecated in https://www.tiny.cloud/docs/tinymce/latest/6.2-release-notes/#deprecated[{productname} 6.2] and has been removed in {productname} 7.0. Instead, use the `trigger` property to specify the `string` that will trigger the autocompleter.
> 
> If you previously used `+editor.ui.registry.addAutocompleter(name, options)+` in your configuration, then update your configuration from `ch: '<string>',` to `trigger: '<string>',`.

Reviewer: @vpyshnenko Can you verify the below

```suggestion
This property was deprecated in link:https://www.tiny.cloud/docs/tinymce/6/6.2-release-notes/#deprecated[{productname} 6.2] and has been removed in {productname} 7.0. Instead, use the `trigger` property to specify the `string` that will trigger the autocompleter.

If `+editor.ui.registry.addAutocompleter(name, options)+` was used  in your configuration, updating your configuration from `ch: '<string>',` to `trigger: '<string>',` is required.
```
